src/main/java/org/mdbenefits/app/submission/actions/ReformatHouseholdNameBeforeSaving.java

```diff
@@ -0,0 +1,30 @@
+package org.mdbenefits.app.submission.actions;
+
+import formflow.library.config.submission.Action;
+import formflow.library.data.FormSubmission;
+import formflow.library.data.Submission;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ReformatHouseholdNameBeforeSaving implements Action {
+    @Override
+    public void run(Submission submission, String id) {
+        ArrayList<HashMap<String, Object>> householdSubflow = (ArrayList<HashMap<String, Object>>) submission.getInputData().get("household");
+        if (householdSubflow != null) {
+            HashMap<String, Object> householdMember = householdSubflow.stream()
+                .filter(iteration -> id.equals(iteration.get("uuid"))).toList().get(0);
+            String firstName = (String) householdMember.remove("householdMemberFirstName");
+            String middleName = (String) householdMember.remove("householdMemberMiddleName");
+            String lastName = (String) householdMember.remove("householdMemberLastName");
+
+            householdMember.put("householdMemberFirstName", firstName.trim());
```

Comment:
I think you can just do a `put` w/o the remove, as it should overwrite the original entry. 
This is good for now, but ultimately, it seems like this is something FFL should do by default. Maybe someday we can get something like this in the library.